### ISSUE TITLE
Accessibility: PDFs rendered with DOMPDF are missing Tags

### ISSUE THREAD

**Issue by WestphalDenn** [dompdf/dompdf#2774]
Hi, 

first of all thank you for all your hard work. 
I am a blind user of Bookstack which uses DOMPDF to export Pages to PDF. When doing so I discovered that the files I have exported are missing tags. 

## Why is that important? 

Tags in PDF files allow asistive technologies such as Screenreaders to make out the structure of a document. e.g. headings and subheadings. If the tags are missing, it is very hard to keep an overview of the document while using assistive technologies. 

## What is happening? 

* I create a Page in Bookstack including headings to provide a document structure. 
* * I export the Page as PDF 
* * The resulting PDF file is missing the heading structure
## Fix: 

Fixing this will allow even more people using DOMPDF and future-proofs it

If Headings, Alt-Text etc. are used in the source-file it should reflect in the PDF as well. 

Thanks again for your hard work and have a nice day.

**Comment by TautvydasPocius** [comment]
Hello, is there any updates on the implementation of this functionality?

**Comment by bsweeney** [comment]
not at this time

**Comment by nebrot** [comment]
First of all, thank you for your continuous work and dedication to this great library!

As the [European Accessibility Act](https://ec.europa.eu/social/main.jsp?catId=1202&intPageId=5581&langId=en) (EAA) comes into effect on June 28, 2025, websites and digital content in the EU will be required to meet accessibility standards, including the generation of accessible PDFs. Currently dompdf does not support the creation of accessible PDFs (tagged PDFs, etc.).

I would like to kindly ask if there are any plans or updates regarding the implementation of accessibility features in dompdf? This would be a crucial feature for those of us working with European clients and aiming to ensure compliance with the upcoming regulations.

Related to #3259

**Comment by bsweeney** [comment]
I'll review to see exactly what would be needed to make this possible.

**Comment by joonlabs** [comment]
@bsweeney Any news so far?

**Comment by nicolasgalvez** [comment]
I just spent a few hours failing to get tags and tabbed content working outside of Acrobat with open source tools. My hat goes off to all the people who work on this project. I wish I knew more about it so I could help with developing the accessibility features. Well I download the PDF spec and will try to make time to go over it, maybe in like a year or two I'll know enough to be of use haha

**Comment by dimablack** [comment]
Any updates?

**Comment by bsweeney** [comment]
Nothing to report, yet.
